core/os/input_event.cpp

```diff
@@ -49,11 +49,11 @@ bool InputEvent::is_action(const StringName &p_action) const {
 	return InputMap::get_singleton()->event_is_action(Ref<InputEvent>((InputEvent *)this), p_action);
 }
 
-bool InputEvent::is_action_pressed(const StringName &p_action) const {
+bool InputEvent::is_action_pressed(const StringName &p_action, bool p_allow_echo) const {
```

Comment:
Looks good, but you should likely register the default value into _bind_methods using DEFVAL. [At line 115.](https://github.com/godotengine/godot/blob/a33fc2a5994a5df711a8e2c881d6ac31b37a9fa6/core/os/input_event.cpp#L115)
This is necessary when generating the documentation.

Reply:
Done.